src/Tools/Source/RunTests/Options.cs

```diff
@@ -58,6 +58,11 @@ internal class Options
         /// </summary>
         public List<string> Assemblies { get; set; }
 
+        /// <summary>
+        /// Time after which the runner should kill the xunit process and exit with a failure.
+        /// </summary>
+        public TimeSpan? Timeout { get; set; }
```

Comment:
Should this just default to TimeSpan.InfiniteTimeSpan to make the null checking go away?